scripts/doc.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+typedoc && touch docs/.nojekyll
```

Comment:
I thought TypeDoc creates a `.nojekyll` file by default.

Reply:
It does! Good catch. Fixed in [750ac80](https://github.com/MetaMask/controllers/pull/831/commits/750ac803401580113f4378a1d0289da6fae55e88).